### PR TITLE
added GetParameters REST method

### PR DESCRIPTION
**Pull request by thomasstorm**

### CODE REVIEW

**Comment by thomasstorm** [comment]
Yet unclear where to put (test) resources, as the json file we simply put on top level.

File: multiply_ui/server/context.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 from typing import Optional, List
 
 from .model import Job
+from .controllers.controller import Controller
 
 
 class ServiceContext:
     def __init__(self):
         self._jobs = {}
+        self.controller = Controller()
```
Comment (forman) [inline multiply_ui/server/context.py]:
```suggestion
        self._controller = Controller()
```

File: multiply_ui/server/controllers/controller.py [hunk 2]
```diff
@@ -3,22 +3,6 @@
...
-
-    def __init__(self):
-        """ Virtually private constructor. """
-        if Controller.__instance is not None:
-            raise Exception("This class is a singleton!")
-        else:
-            Controller.__instance = self
-
     def get_parameters(self):
         with open('test/test_data/example_parameters.json') as f:
```
Comment (forman) [inline multiply_ui/server/controllers/controller.py]: Path changed, see my screenshot in Skype!
Don't use file paths, they wont work after packaging, instead do
```
import pkg_resources
json_text = pkg_resources.resource_string("multiply_ui", "server/resources/processing-parameters.json")
return json.loads(json_text)
```

File: multiply_ui/server/context.py (hunk 1 above)
Comment (forman) [reply multiply_ui/server/context.py]: Actually there is no need for this. Controllers can be plain Python function with first parameter being the context passed from handlers.

**Review by forman — CHANGES_REQUESTED** [review]

File: multiply_ui/server/handlers.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import tornado.web
 
 from .context import ServiceContext
+from .controllers import controller
```
Comment (forman) [inline multiply_ui/server/handlers.py]: Ok, dann könntet ihr auch das package `multiply_ui/server/controllers` auflösen und stattdessen nur mit dem modul `multiply_ui/server/controller.py` arbeiten, ne?

**Review by forman — APPROVED** [review]

**Review by TonioF — APPROVED** [review]

**Pull request merged**